packages/jwto/jwto.0.3.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+authors: "Sebastian Porto"
+maintainer: "Sebastian Porto <s@porto5.com>"
+license: "MIT"
+homepage: "https://github.com/sporto/jwto"
+bug-reports: "https://github.com/sporto/jwto"
+doc: "https://sporto.github.io/jwto/"
+dev-repo: "git+https://github.com/sporto/jwto.git"
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune" {build}
```

Comment:
Dune is not compatible with the `{build}` tag. Furthermore this package requires at least dune 1.7 according to your `dune-project` file. Could fix that?